modules/custom-status/meta/position-handler.php

```diff
@@ -29,9 +29,9 @@ public static function init(): void {
 	 * @return array The updated meta keys
 	 */
 	public static function add_position( array $term_meta, WP_Term $custom_status ): array {
-		$position = MetaCleanupUtilities::get_int( $custom_status->term_id, Custom_Status::METADATA_POSITION_KEY );
+		$position = MetaCleanupUtilities::get_int( $custom_status->term_id, CustomStatus::METADATA_POSITION_KEY );
```

Comment:
I like that you've split these filter side-effects out into a new file. My ideal next step would be to move the `METADATA_POSITION_KEY` constant into this file, and also add a `Position::get()` and `Position::set()` wrappers so `CustomStatus` doesn't own or need to know the details about how we store values on a term, and can just call into these meta files.

Reply:
Agreed, I didn't do that yet as it's going to require testing and planning to ensure it doesn't cause any conflicts. I also wanted to use that as a chance to make sure proper GET/GET ALL were in place so those became the default entry points for everything instead of the individual methods.